src/components/standalone/firewall/CreateOrEditPortForwardDrawer.vue

```diff
@@ -286,13 +298,17 @@ async function createOrEditPortForward() {
     "
   >
     <NeInlineNotification
-      v-if="error.notificationTitle"
-      :title="error.notificationTitle"
-      :description="error.notificationDescription"
+      v-if="error.notificationTitle || firewallConfig.error"
+      :title="firewallConfig.error ? t('error.cannot_retrieve_zones') : error.notificationTitle"
+      :description="
+        firewallConfig.error
+          ? t(getAxiosErrorMessage(firewallConfig.error))
+          : error.notificationDescription
+      "
       class="mb-6"
       kind="error"
     />
-    <NeSkeleton v-if="loading || firewallConfig.loading" :lines="10" />
+    <NeSkeleton v-if="loading || firewallConfig.loading || firewallConfig.error" :lines="10" />
```

Comment:
We usually don't show a skeleton if an error is displayed

Reply:
Hmm, this is true but doing it this way could also cause a scenario where the user can interact with the form even if the firewallConfig fetch did not go as planned... How should we handle this case?

Reply:
> Hmm, this is true but doing it this way could also cause a scenario where the user can interact with the form even if the firewallConfig fetch did not go as planned... How should we handle this case?

If an error occurs while retrieving drawer data we usually display only the error and hide the form

Reply:
Ok, now if there's an error while fetching the firewall config the form does not show. I've also removed the condition from the skeleton.